settings.gradle

```diff
@@ -0,0 +1,4 @@
+def mSearchDir = new File(settingsDir, '../MSearch')
```

Comment:
MSearch landet derzeit noch in `../mSearch`, da das von `ant` so erwartet wird/wurde, das solltest du noch anpassen.